frontend/app-development/components/WebSocketEntityUpdateWrapper/WebSocketEntityUpdateWrapper.test.tsx

```
@@ -0,0 +1,97 @@
+      webSocketConnector: WSConnector,
+    });
+  });
+
+  it('should invoke onWSMessageReceived when EntityUpdated message is received', async () => {
+    const entityUpdatedMock: EntityUpdated = {
+      resourceName: 'Deployment', // Example resource name for entity update
+    };
+
+    const mockOnWSMessageReceived = jest
+      .fn()
+      .mockImplementation((callback: Function) => callback(entityUpdatedMock));
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)